scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionAnomaly.scala

```diff
@@ -47,16 +48,15 @@ trait H2OMOJOPredictionAnomaly {
     Seq(StructField(getPredictionCol(), predictionColType, nullable = predictionColNullable))
   }
 
-  def getAnomalyDetailedPredictionColSchema(): Seq[StructField] = {
+  def getAnomalyPredictionSchema(): StructType = {
     val scoreField = StructField("score", predictionColType, nullable = false)
     val fields = if (getWithDetailedPredictionCol()) {
       val normalizedScoreField = StructField("normalizedScore", predictionColType, nullable = false)
       scoreField :: normalizedScoreField :: Nil
     } else {
       scoreField :: Nil
     }
-    Seq(StructField(getDetailedPredictionCol(), StructType(fields), nullable = true))
-
+    StructType(fields)
```

Comment:
`object H2OMOJOPredictionAnomaly` with the defined case classes can be removed

Reply:
Good catch, thanks!